### PR TITLE
Update for new modloader and HarmonyX. (Not tested, just fixed all errors)

### PR DESCRIPTION
**Pull request by lovhb**
Updated harmony and added the new modloader framework. Fixed all errors. Haven't tested though as I don't have a haptic vest.

### CODE REVIEW

**Comment by MarshMello0** [comment]
This fixes #1

File: .idea/.idea.vtol_bhaptics/.idea/discord.xml
Comment (MarshMello0) [inline .idea/.idea.vtol_bhaptics/.idea/discord.xml:1]: You might want to add `.idea/` to the `.gitignore` as people might have different prefences to these settings

File: vtol_bhaptics/vtol_bhaptics.csproj [hunk 1]
```diff
@@ -31,30 +31,38 @@
...
     <Reference Include="UnityEngine">
-      <HintPath>..\Dependencies\UnityEngine.dll</HintPath>
+      <HintPath>C:\Program Files (x86)\Steam\steamapps\common\VTOL VR\VTOLVR_Data\Managed\UnityEngine.dll</HintPath>
     </Reference>
     <Reference Include="UnityEngine.CoreModule">
-      <HintPath>..\Dependencies\UnityEngine.CoreModule.dll</HintPath>
+      <HintPath>C:\Program Files (x86)\Steam\steamapps\common\VTOL VR\VTOLVR_Data\Managed\UnityEngine.CoreModule.dll</HintPath>
     </Reference>
     <Reference Include="UnityEngine.PhysicsModule">
-      <HintPath>..\Dependencies\UnityEngine.PhysicsModule.dll</HintPath>
```
Comment (MarshMello0) [inline vtol_bhaptics/vtol_bhaptics.csproj]: You also might want to change these back as not everyone has the game installed on their C drive. But everyone can make mklinks from the Dependencies folder to their game folder and not have to commit over each other. 😄

Comment (lovhb) [reply vtol_bhaptics/vtol_bhaptics.csproj]: Oh i didn't realize rider uses absolute file paths for references... will fix

**Comment by lovhb** [comment]
Update: The mod now works in game. The only remaining issue is that gun_fire rumble is apparently slightly delayed.

**Comment by McFredward** [comment]
Just tested it - it's working. Thanks!
This community is so nice - the problem just got fixed before I even had time to look at it haha.

Will try to fix some old bugs and bugs which appeared because of the games updates the next days.

**Pull request merged**